### ISSUE TITLE
Method ambiguities reported by Aqua

### ISSUE THREAD

**Issue by prbzrg** [JuliaCollections/SortingAlgorithms.jl#90]
In Julia v1.10, I used Aqua to detect method ambiguities in my package https://github.com/impICNF/ContinuousNormalizingFlows.jl/issues/356 and some of its report were related to this package:

```julia
Ambiguity #105
sort!(vs::AbstractVector{T}, lo::Int64, hi::Int64, ::SortingAlgorithms.RadixSortAlg, o::Base.Order.Ordering, ts::Union{Nothing, AbstractVector{T}}) where T @ SortingAlgorithms ~/.julia/packages/SortingAlgorithms/3sO8j/src/SortingAlgorithms.jl:582
sort!(v::AbstractVector, lo::Integer, hi::Integer, a::Base.Sort.Algorithm, o::Base.Order.Ordering, scratch::Vector) @ Base.Sort sort.jl:2209

Possible fix, define
  sort!(::AbstractVector{T}, ::Int64, ::Int64, ::SortingAlgorithms.RadixSortAlg, ::Base.Order.Ordering, ::Vector{T}) where T


```

I would appreciate your attention to this issue.

**Comment by LilithHafner** [comment]
Thank you for reporting. This was fixed in #80 but I haven't released it yet because the ambiguity does not effect users of this package's public API or Base's public API. It could be considered either bad API design or an Aqua false positive, either way it's easy to fix and has been fixed. I will release 1.2.1 with the fix for this presently.

**Comment by LilithHafner** [comment]
Should be fixed in the latest release
